HISTORY.rst

```diff
@@ -7,6 +7,7 @@ Unreleased
 
 * `glinter` messages have been improved with more details and to be more
   actionable.
+* Swift: Combine all metrics and pings into a single generated file `Metrics.swift`.
```

Comment:
Should this be flagged as a breaking change, since it changes who users need to import their metrics?  (If I'm right about that...)

Reply:
Ah yes, true indeed.
Once we upgrade the parser in the SDK we should make sure to change the docs